apps/system/utils/fscmd.c

```diff
@@ -273,7 +273,9 @@ static int tash_cat(int argc, char **args)
 	int destfd;
 	int i;
 	int flags;
-	ssize_t ret = 0;
+	ssize_t read_size;
+	ssize_t written_size = 0;
+	int ret;
```

Comment:
How about initializing the `ret` as `ERROR` and removing line 317, 356, 363 and 370?

Reply:
Thank you for suggestion. But current way is easier to know the failure and does not get affected from code change.  It will make us check all of code to check whether it is set well.